mods/cnc/rules/defaults.yaml

```diff
@@ -180,6 +180,7 @@
 			Beach: 80
 	SelectionDecorations:
 	Selectable:
+		Bounds: 12,17,0,-6
```

Comment:
No, TD infantry is smaller. The difference is more than 1 pixel actually, but I didn't feel like scaling 100% accurately because otherwise either TD Bounds would end up too small (making it harder to select them) or RA/D2k bounds too large (risking overlapping)